a11y-meta-display-guide/2.1/draft/techniques/onix-metadata/index.html

```diff
@@ -1058,11 +1064,7 @@ <h4 id="accessibility-summary-instructions">Instructions</h4>
 					</li>
 					<li>
 						<span><b>IF</b> <var>publisher_contact_for_accessibility</var> is <b>NOT</b> empty:</span>
-						<span><b>THEN</b></span>
-						<ol class="display">
-							<li>display <code id="accessibility-summary-publisher-contact">"For more information about the accessibility of this product, please contact the publisher: "</code></li>
-							<li>display <var>publisher_contact_for_accessibility</var> as link <b>IF</b> <var>publisher_contact_for_accessibility</var> is an email address.</li>
-						</ol>
+						<span><b>THEN</b> display <code id="accessibility-summary-publisher-contact">"For more information about the accessibility of this product, please contact the publisher: {publisher_contact_for_accessibility}"</code> replacing <code>"{publisher_contact_for_accessibility}"</code> with <var>publisher_contact_for_accessibility</var> as link <b>IF</b> <var>publisher_contact_for_accessibility</var> is an email address.</span>
```

Comment:
It should be obvious, but it might be better to say to replace with a "mailto link" here.
```suggestion
						<span><b>THEN</b> display <code id="accessibility-summary-publisher-contact">"For more information about the accessibility of this product, please contact the publisher: {publisher_contact_for_accessibility}"</code> replacing <code>"{publisher_contact_for_accessibility}"</code> with <var>publisher_contact_for_accessibility</var> as a mailto link <b>IF</b> <var>publisher_contact_for_accessibility</var> is an email address.</span>
```

Reply:
Also, can the last "IF ... is an email address." be dropped? Doesn't onix require an email address?